src/pdbpp.py

```diff
@@ -134,6 +144,10 @@ class DefaultConfig(object):
     line_number_color = Color.turquoise
     filename_color = Color.yellow
     current_line_color = "39;49;7"  # default fg, bg, inversed
+    # Windows/colorama doesn't support 'inversed', so instead of using the
+    # terminal default colors, we just use black-on-white
+    if sys.platform == "win32":
+        current_line_color = "30;47"  # black, white
```

Comment:
Does `49;39` work / make sense?  (poor man's inversed)

Reply:
Sadly no, that doesn't make sense. The codes are independent of the location within the escape sequence. These are all the same "bold yellow text on blue background":
![image](https://user-images.githubusercontent.com/5386897/73961331-38f9bb80-48c1-11ea-937e-37827f6a6313.png)

With `49;39`:
![image](https://user-images.githubusercontent.com/5386897/73960459-ccca8800-48bf-11ea-8e13-c6b608131d81.png)

With `30;47`:
![image](https://user-images.githubusercontent.com/5386897/73960512-e9ff5680-48bf-11ea-81b5-5986f2d7b10c.png)

**If** there was a way to detect the current terminal's default background and foreground colors, then I could manually invert them. However, I'm not aware of any way to do that. It *may* be possible on Linux using [`curses`](https://docs.python.org/3/library/curses.html#curses.use_default_colors) (see [this SO answer](https://stackoverflow.com/a/44015131/1354930)), but I can't find anything on how to get the default colors in Windows CMD/PowerShell.

Reply:
I've thought since `ESC[39;49m` sets/uses the default fg/bg this could be reversed maybe (just a silly thought.. :D)
As for what curses does (the SO answer) I guess it just skips setting it then, but have not checked actually.

I guess it is fine to use black on white then - although that might only work for dark schemes really.

After all it should / could use colors from the pygments theme/formatter maybe?